src/components/Personalization/flicker/index.js

```diff
@@ -10,8 +10,15 @@ OF ANY KIND, either express or implied. See the License for the specific languag
 governing permissions and limitations under the License.
 */
 
-import { createNode, appendNode, removeNode } from "../../../utils/dom";
+import {
+  createNode,
+  appendNode,
+  removeNode,
+  findById
+} from "../../../utils/dom";
 
+const CONTAINERS_HIDING_ID = "alloy-prehiding";
+const HIDING_STYLE_DEFINITION = "{ visibility: hidden }";
 const STYLE_TAG = "style";
 const CACHE = {};
```

Comment:
This global state scared me at first.  From a functionality perspective I understand why it is necessary even with multiple instances on a page; namely, the DOM is global so the cache should be global too.  Maybe a comment here could explain why the global state is necessary.

Additionally, I think renaming it as `styleNodes` would make it more clear as to what it is doing.  My understanding is that variables named in all caps should stay constant.

Reply:
Fixed